DEV_SETUP.md

```diff
@@ -64,33 +64,23 @@ NOTE: Developers on Mac OS have additional prerequisites. See the [Supplementary
     brew install python@3.9
     ```
 
-- A Python virtual environment manager like `pyenv` or `virtualenvwrapper`. We recommend `pyenv` ([see installation guide](https://github.com/pyenv/pyenv#installation))
-
-    To install python 3.9 with `pyenv`:
 
-    ```sh
-    pyenv install 3.9.11
-    ```
-
-    To set Python 3.9 as the global `python`, run:
-    ```sh
-    pyenv global 3.9.11
-    ```
-    Pro-tip: this is great for Mac OS users working around having to explicitly use `python3` :)
-
-
-- Requirements of Python libraries, if they aren't already installed.
+- Requirements of Python libraries, if they aren't already installed.  Some of this comes from
+  [pyenv's recommendations](https://github.com/pyenv/pyenv/wiki#suggested-build-environment)
 
   - **Linux**:
 
     ```sh
-    sudo apt install libncurses-dev libxml2-dev libxmlsec1-dev libxmlsec1-openssl libxslt1-dev libpq-dev pkg-config gettext
+    sudo apt-get update; sudo apt install libncurses-dev libxml2-dev libxmlsec1-dev \
+    libxmlsec1-openssl libxslt1-dev libpq-dev pkg-config gettext make build-essential \
+    libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev
```

Comment:
I wonder if we need `tk-dev`? Probably not an issue on most developer workstations since they probably have X11 installed, but I think that pulls in some pretty big dependencies (like X11).

Reply:
No idea, I pulled this from `pyenv`'s ["recommendations for a sane build environment"](https://github.com/pyenv/pyenv/wiki#suggested-build-environment).

My preference for this doc is heavily in favor of simplicity and reliability, so I'm inclined to leave it in (especially versus adding another "troubleshooting" section), but I wouldn't object to its removal if you're pretty confident we can do without.  We add wording around these being recommendations that might not all be necessary, allowing ambitious setter-uppers to fiddle, while I expect most would just copy/paste.

Reply:
Sure, seems reasonable to leave as is.